docs/source-pytorch/versioning.rst

```diff
@@ -81,6 +81,12 @@ The table below indicates the coverage of tested versions in our CI. Versions ou
      - ``torch``
      - ``torchmetrics``
      - Python
+   * - 2.2
+     - 2.2
+     - 2.2
+     - ≥1.13, ≤2.2
+     - ≥0.7.0
+     - ≥3.8, ≤3.11
```

Comment:
```suggestion
```
This is not part of this PR, right?

Reply:
Why not? We want to document with which versions of PyTorch Lightning 2.2 is compatible with.

Reply:
Yes, I am just saying it shall be in the other PR =)